src/utils.ts

```diff
@@ -197,3 +199,22 @@ export async function getCurrentFileSelectionPath(): Promise<vscode.Uri> {
 export function setContext(inContext: vscode.ExtensionContext) {
 	context = inContext;
 }
+
+export function getLastColumnUsedSetting() : number {
```

Comment:
the method is not getting only the lastColumnUSedSetting. In case it doesn't exist, it is using other heurisitic.

A better naming might be getColumn? or getLastColumn?